apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json

```diff
@@ -0,0 +1,280 @@
+        {
+          "answer": "To facilitate wallet connections"
+        },
+        {
+          "answer": "To log transaction details for minted NFTs"
+        },
+        {
+          "answer": "To handle conditional rendering of NFT cards"
+        },
+        {
+          "answer": "To initiate the minting process for NFTs",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "How is error handling enhanced in the `mintToken` function?",
+      "options": [
+        {
+          "answer": "With a big, red, digital, teacher’s pen"
+        },
+        {
+          "answer": "By using a `catch` statement",
+          "correct": true
+        },
+        {
+          "answer": "By disabling mint buttons if the user doesn’t have enough to mint those tokens"
+        },
+        {
+          "answer": "By slowing down the minting process"
+        }
+      ]
+    },
+    {
+      "question": "How does a modal enhance value for users of a Web3 app?",
+      "options": [
+        {
+          "answer": "It modifies contract state"
+        },
+        {
+          "answer": "It displays contract ABI code"
+        },
+        {
+          "answer": "It provides an overlay for focused interaction",
+          "correct": true
+        },
+        {
+          "answer": "It manages wallet connections"
+        }
+      ]
+    },
+    {
+      "question": "Why do we use the `refetchTokenData` function in the `mintToken` function?",
+      "options": [
+        {
+          "answer": "To display a loading message"
+        },
+        {
+          "answer": "To update the `latestNFTMinted` variable"
+        },
+        {
+          "answer": "To update the `tokenData` after minting has succeeded",
+          "correct": true
+        },
+        {
+          "answer": "To verify user connection"
+        }
+      ]
+    },
+    {
+      "question": "What triggers the display of the modal in the `mintToken` function?",
+      "options": [
+        {
+          "answer": "Setting `isMinting` to true"
+        },
+        {
+          "answer": "Setting `modalShow` to true",
+          "correct": true
+        },
+        {
+          "answer": "Updating the `latestNFTMinted` variable"
+        }
+      ]
+    },
+    {
+      "question": "How come we can read the value of a Solidity variable on our front-end?",
+      "options": [
+        {
+          "answer": "The `useEffect` hook updates a string to fetch it"
+        },
+        {
+          "answer": "When the value of `totalSupply` equals `tokenID`, it becomes possible"
+        },
+        {
+          "answer": "The variable is defined with `external` visibility"
+        },
+        {
+          "answer": "The variable is defined with the `public` keyword",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "Which of the following shows on our modal footer?",
+      "options": [
+        {
+          "answer": "Join Developer_DAO"
```

Comment:
Is the underscore an official part of the Developer DAO name?

(I removed it in my Lesson 5 PR... lemme know if I should add it back)